File: app/src/main/java/com/example/android/moviesremake/retrofit/response/ReviewResponse.java
package com.example.android.moviesremake.retrofit.response;

import com.example.android.moviesremake.retrofit.model.MovieRetrofitReview;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by PepovPC on 1/17/2018.
 */

public class ReviewResponse {


    @SerializedName("results")
    private List<MovieRetrofitReview> resultsReview;

    public List<MovieRetrofitReview> getResultsForReview() {
        return resultsReview;
    }


}

File: app/src/main/java/com/example/android/moviesremake/retrofit/response/ReviewResponse.kt
package com.example.android.moviesremake.retrofit.response

import com.example.android.moviesremake.retrofit.model.MovieRetrofitReview
import com.google.gson.annotations.SerializedName

/**
 * Created by PepovPC on 1/17/2018.
 */

class ReviewResponse {


    @SerializedName("results")
    val resultsForReview: List<MovieRetrofitReview>? = null


}